### ISSUE TITLE
Not clear how to use

### ISSUE THREAD

**Issue by jagiella** [alangrafu/visualRDF#10]
Your Java script looks really great and I would like to use it. Unfortunately, it is not clear to me what I have to pass: I have locally a SPARQL server (Jene Fukesi apache) installed. And I passed its URL to your testing HTML:
1. http://http://graves.cl/visualRDF/?url=http://localhost:3030/Proteins
   -> http://localhost:3030/Proteins returns the RDF in Turle format
2. http://http://graves.cl/visualRDF/?url=http://localhost:3030/Proteins/query
   -> http://localhost:3030/Proteins/query answers SPARQL queries
3. http://graves.cl/visualRDF/?url=file:///C:/Users/njagiella/Downloads/Proteins.rdf
   -> file:///C:/Users/njagiella/Downloads/Proteins.rdf is my local RDF file in XML format

Any idea why it doesn't process my onthology?

**Comment by alangrafu** [comment]
Hi, 

There are a couple of issues: First, you are passing a localhost URI to graves.cl. graves.cl CANNOT access your localhost, that's your local machine.

Once you fix that, visualRDF will accept any URI containing a RDF document in RDF/XML, turtle or ntriples.

**Comment by jagiella** [comment]
Hi thank you for your fast answer!

I guessed that this might be a problem. As I understand your test page, it loads some Java scripts which do the job of visualizing the RDF graphs. So, if it doesn't work with localhosts, it's due to passing the URL (of the RDF) to an external page then?

I tried as well an openly accessible onthology server, but had no success either:

http://graves.cl/visualRDF/?url=http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl

Any clue, why it doesn't work?
